GCRCatalogs/alphaq.py

```diff
@@ -68,8 +68,8 @@ def _subclass_init(self, filename, **kwargs):
             'stellar_mass':       'totalMassStellar',
             'size_disk_true':     'morphology/diskHalfLightRadiusArcsec',
             'size_bulge_true':    'morphology/spheroidHalfLightRadiusArcsec',
-            'disk_Sersic_index':  'morphology/diskSersicIndex',
-            'bulge_Sersic_index': 'morphology/spheroidSersicIndex',
+            'disk_Sersic':  'morphology/diskSersicIndex',
```

Comment:
should be `sersic_disk` (lower case, component name should come after quantity name)

Reply:
This google doc has 'disk_Sersic_index' https://docs.google.com/document/d/1rUsImkBkjjw82Xa_-3a8VMV6K9aYJ8mXioaRhz0JoqI/edit#heading=h.9djco1snvvi3

Is there another place with the schema?